mlir/lib/Dialect/Linalg/Transforms/TilingInterfaceImpl.cpp

```diff
@@ -132,6 +132,59 @@ struct LinalgOpTilingInterface
     return TilingResult{{tiledOp}, SmallVector<Value>(tiledOp->getResults())};
   }
 
+  void getMappedOffsetAndSize(Operation *op, OpBuilder &b,
```

Comment:
Nit: just make the operand `LinalgOp linalgOp` type instead of `Operation *op`. You can then do 

```
auto tilingInterfaceOp = cast<TilingInterface>(linalgOp.getOperation());
```

Reply:
Thanks for your comments! It's crunch week for me now, I'll address your review comments ASAP.

Reply:
DONE